### PR TITLE
Simple patient list

### PR DESCRIPTION
**Pull request by CarlosNZ**
_Copied from old repo: https://github.com/openmsupply/openmsupply-client/pull/1255 from @mark-prins_ 

Fixes openmsupply/open-msupply#1 

A list view for patients.. just something to get us started.
Currently using the name API which doesn't return much patient info, but is a useful placeholder. Have added this at the top level in the navigation for now, which causes a bit of an issue with how we handle the breadcrumb and page title. Something to consider.

### CODE REVIEW

**Comment by CarlosNZ** [comment]
Migrated to new monorepo: https://github.com/openmsupply/open-msupply/pull/3

File: client/packages/host/src/routers/PatientRouter.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const fullPatientsPath = RouteBuilder.create(AppRoute.Patients)
+  .addWildCard()
+  .build();
+
+export const PatientRouter: FC = () => {
+  const gotoPatients = useMatch(fullPatientsPath);
+
+  if (gotoPatients) {
+    console.log('GOOO');
```
Comment (clemens-msupply) [inline client/packages/host/src/routers/PatientRouter.tsx:17]: Debug output?

File: client/packages/system/src/Patient/DetailModal/DetailModal.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    <DetailContainer>
+      <Box display="flex" flexDirection="column" alignItems="center" gap={2}>
+        <Typography sx={{ fontSize: 18, fontWeight: 700 }}>
+          {data.name}
+        </Typography>
+        <Grid container flex={1} flexDirection="row" gap={4}>
+          <DetailSection title="">
+            <DetailInputWithLabelRow
+              label={t('label.address')}
+              inputProps={{ value: data?.address, disabled: isDisabled }}
```
Comment (clemens-msupply) [inline client/packages/system/src/Patient/DetailModal/DetailModal.tsx:45]: I fleshed out the name sync a bit and broke this code while doing so :(

There is now address1 and address2 (was billingAddress in mSupply)...

**Review by clemens-msupply — APPROVED** [review]
Thanks! I still got caught up with name sync and will work on a patient API now. There are some changes in the name details that already have been merged during the repo merge, e.g. address -> address1...

File: client/packages/host/src/routers/PatientRouter.tsx (hunk 1 above)
Comment (mark-prins) [reply client/packages/host/src/routers/PatientRouter.tsx:17]: 🤦

File: client/packages/system/src/Patient/api/api.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      const result = await sdk.names({
+        first,
+        offset,
+        key,
+        desc: !!sortBy?.isDesc,
+        storeId,
+        filter: {
+          isCustomer: true,
+        },
+      });
```
Comment (CarlosNZ) [inline client/packages/system/src/Patient/api/api.ts:45]: Should we make this `isCustomer: false`?

**Review by CarlosNZ — COMMENTED** [review]
Looks like a good skeleton, thanks. @mark-prins I stole this PR from you when I migrated it from the old repo, so please merge it when you're ready.

File: client/packages/system/src/Patient/api/api.ts (hunk 3 above)
Comment (mark-prins) [reply client/packages/system/src/Patient/api/api.ts:45]: when I created the PR, there was no patient filter ( still isn't ) and no type to distinguish patients.
This is a temporary filter, to get some results to display

**Comment by mark-prins** [comment]
Updated to cope with the schema changes, and renamed a little to get the codegen working 😉

**Pull request merged**